CustomApps/new-releases/Card.js

```diff
@@ -19,6 +19,11 @@ class Card extends react.Component {
 		event.stopPropagation();
 	}
 
+	closeButtonClicked(event) {
+		removeCards(this.props.uri);
+		event.stopPropagation();
```

Comment:
> this also means if it errors it just opens the cards page instead of crashing spotify.

Execution halts when an exception is thrown, so that doesn't help with anything.

Reply:
We already established that this was meant to not open the page after clicking the dismiss button.